sbt-app/src/sbt-test/project-matrix/plugin/build.sbt

```
@@ -0,0 +1,28 @@
+val scala3 = "3.3.3"
+val scala212 = "2.12.19"
+
+organization := "com.example"
+version := "0.1.0-SNAPSHOT"
+
+lazy val checkSbt2Plugin = taskKey[Unit]("")
+
+lazy val plugin = (projectMatrix in file("plugin"))
```

It seems Scala 3 is going towards less [infix notation](https://docs.scala-lang.org/scala3/reference/changed-features/operators.html). Should we agree not to use it here anymore?

```suggestion
lazy val plugin = projectMatrix.in(file("plugin"))
```

---

I'm generally +1 on most Scala 3 directions including limiting the use of infixes. However, `in` is clearly intended to act like an English DSL / pretend-keyword so we should actually keep it as such, perhaps by marking it infix.